.circleci/config.yml

```diff
@@ -1,52 +1,65 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
 jobs:
-  build:
-    machine: true
-    #support this after we can enable it on account
-    #docker_layer_caching: true
+  test_centos7:
+    machine:
```

Comment:
be warned, provisioning "machine" over and over actually allocates a full vm to us.  We pay for a plan based on the number of those that run.  We're good if people run this single threaded, but if parallelism gets added, this will really drive up our costs.